src/main/java/org/akhq/controllers/TopicController.java

```diff
@@ -192,6 +192,12 @@ public Topic home(String cluster, String topicName) throws ExecutionException, I
         return this.topicRepository.findByName(cluster, topicName);
     }
 
+    @Get("api/{cluster}/topic/{topicName}/last-record")
+    @Operation(tags = {"topic"}, summary = "Retrieve the last record of a topic")
+    public Record lastRecord(String cluster, String topicName) throws ExecutionException, InterruptedException {
+        return this.recordRepository.getLastRecord(cluster, topicName).orElse(null);
```

Comment:
```suggestion
        return this.recordRepository.getLastRecord(cluster, topicName).orElse(new Record());
```
To avoid 404 on topic with no data